Lilu/Headers/kern_disasm.hpp

```diff
@@ -54,7 +61,14 @@ class Disassembler {
 	EXPORT static size_t quickInstructionSize(mach_vm_address_t ptr, size_t min);
 
 	/* Note, code should point to at least 32 valid bytes. */
+#if defined(__i386__)
+	EXPORT static size_t hdeDisasm(mach_vm_address_t code, hde32s *hs);
```

Comment:
In my opinion, go with `using` to choose the local struct.

Reply:
Corrected in https://github.com/acidanthera/Lilu/commit/5041831557fbde00ab343a608614a180b4ee0f62